tests/spec/test-run-api-service.js

```diff
@@ -258,6 +275,20 @@
                 server.requests = [];
 
             });
+            it('should not include the AutoRestore header', function () {
+                var rs = new RunService({ account: 'forio', project: 'js-libs' });
+                rs.filter({ saved: true, '.price': '1' });
+
+                var req = server.requests.pop();
+                req.requestHeaders.should.not.have.property('X-AutoRestore');
+            });
+            it('should include the AutoRestore header', function () {
+                var rs = new RunService({ account: 'forio', project: 'js-libs' });
+                rs.filter('myfancyrunid');
+
+                var req = server.requests.pop();
```

Comment:
Add another test-case to show that it's respecting the config option to not add the header